crates/bevy_ecs/src/bundle.rs

```diff
@@ -309,16 +356,21 @@ macro_rules! tuple_impl {
         )]
         $(#[$meta])*
         impl<$($name: Bundle),*> DynamicBundle for ($($name,)*) {
+            type Effect = ($($name::Effect,)*);
+            #[allow(
```

Comment:
You might need to expect allow here to get CI to pass.

Reply:
Its already defined on the trait, which allows it here too.